### PR TITLE
add icon for custom sections

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#283

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/context/admin-org.js [hunk 1]
```diff
@@ -50,13 +56,53 @@ const AdminOrgProvider = (props) => {
...
+  };
+
   const value = {
     adminOrg,
     getAdminOrgInfo,
     setAdminOrg: _setAdminOrg,
   };
 
-  return <AdminContext.Provider value={value} {...props} />;
+  // return <AdminContext.Provider value={value} {...props} />;
```
Comment (hassanelnajjar) [inline client/src/context/admin-org.js:95]: please remove this comment

File: client/src/pages/Admin/SectionForm/index.js [hunk 2]
```diff
@@ -340,6 +344,10 @@ const SectionForm = ({ review }) => {
             handleChange={(input) => setState({ title: input })}
           />
         </Col>
+
+        <Col w={[4, 6, 4]}>
```
Comment (hassanelnajjar) [inline client/src/pages/Admin/SectionForm/index.js:348]: Can you change the Number of columns for the whole section (title plus the icon) to be (4,12,6) to match the other fileds.

File: client/src/pages/OrgPublic/Section/index.js [hunk 3]
```diff
@@ -72,7 +72,7 @@ const Section = () => {
     (resource) => resource.key === 'STILL_NEED_HELP'
   );
 
-  const colors = contentColors[id] || contentColors[1];
+  const colors = contentColors[id] || contentColors[sectionData.themeKey || 1];
```
Comment (hassanelnajjar) [inline client/src/pages/OrgPublic/Section/index.js:75]: Can you add (?) on sectionData?.themeKey

**Review by hassanelnajjar — COMMENTED** [review]
Please fix these minor comments.

File: client/src/context/admin-org.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/context/admin-org.js:95]: done

File: client/src/pages/Admin/SectionForm/index.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/pages/Admin/SectionForm/index.js:348]: done

File: client/src/pages/OrgPublic/Section/index.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/pages/OrgPublic/Section/index.js:75]: done

**Review by hassanelnajjar — APPROVED** [review]

**Pull request merged**